monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -136,7 +142,7 @@ def fill_template_config(self, data_stats_filename, **kwargs):
         Args:
             data_stats_filename: filename of the data stats report (generated by DataAnalyzer)
         """
-        pass
+        raise NotImplementedError(f"Subclass {self.__class__.__name__} must implement this method.")
```

Comment:
I think it's nicer to use `pass` to make this method call optional, the bundle template might be ready to train without this template filling..

Reply:
Yes that's possible. Filling the template is optional. The user can simply write a train.py without any configs.

Reply:
Added more doc-strings to explain this behavior